tests/test_mcp_rules.py

```
@@ -648,6 +649,64 @@ def test_valid_headers_on_http_server(temp_dir):
     assert len(violations) == 0
 
 
+def test_editor_mcp_credentials_are_reported_without_echoing_values(temp_dir):
+    cursor_secret = "ghp_A1b2C3d4E5f6G7h8I9j0K1l2M3n4O5p6Q7r8"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "cursor_secret"

(S105)